docs/learn-account-generation.md

```diff
@@ -348,3 +349,28 @@ computer a very small and limited scope for interacting with the Parity Signer d
 continue to keep your key safe.
 
 ![Parity Signer Create Account 6](assets/accounts/ps-create-6.jpg)
+
+## Vanity Generator
+
+Vanity generator is a tool on [Polkadot-JS UI](https://polkadot.js.org/apps/#/accounts/vanity) that
+lets you generate addresses which contain a specific substring.
+
+![Vanity Generator page](assets/polkadot_vanity_generator_page.png)
+
+The vanity generator takes the following parameters:
+
+- "Search for": The substring that you would like to include in your new address.
+- "case sensitive": Yes if the substring will be case sensitive or no if not.
+- "keypair crypto type": Specifies the type of account you'd like to generate, Schnorrkel is
+  recommended in most cases for its securtity.
+
+If you've filled out all details above and hit the "Start generation" button, a long list of
+accounts will start generating on your screen.
```

Comment:
```suggestion
accounts will start generating on your screen.  Note that depending on the length of the substring and the processing speed of your computer, it may take some time before any accounts appear.
```